rdt-open-reader/src/main/assets/json.form-id/patient-registration-form.json

```diff
@@ -1,6 +1,8 @@
 {
-  "count": "1",
```

Comment:
I think a separate form should be created for the covid work and placed in it's own folder.

This form should not be modified.